tests/unit/testing/test_cos.py

```diff
@@ -245,11 +248,50 @@ async def test_check_metrics_endpoint(mock_run_on_unit, metrics_endpoint, exp_cm
         ),
     ],
 )
-def test_get_alert_rules(data, exp_alert_rules):
+def test__get_alert_rules(data, exp_alert_rules):
```

Comment:
is this intentional?

Reply:
Yes, there are two functions `get_alert_rules`  and `_get_alert_rules`. This test is for `_get_alert_rules` and it was skipped due to another test with name `test_get_alert_rules`, which you can see below.